pom.xml

```diff
@@ -411,6 +432,12 @@
         <version>${fr.opensagres.xdocreport.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>xerces</groupId>
+        <artifactId>xercesImpl</artifactId>
+        <version>${xerces.xercesImpl.version}</version>
+      </dependency>
+
       <dependency>
         <groupId>org.subethamail</groupId>
```

Comment:
According to https://github.com/voodoodyne/subethasmtp/tree/master this lib is unmaintained. Maybe it's a good time to drop it and use another one.

Reply:
hi @jendib maybe I can look into it in a separate PR? In this PR i have not touched this library. If you can guide me to the unit tests and functionality that uses this dependency I can work on a different library and see if the test pass and the functionality is preserved after the change.